lib/querystring.js

```diff
@@ -256,6 +255,23 @@ QueryString.parse = QueryString.decode = function(qs, sep, eq, options) {
 };
 
 
+function replacePlus(str) {
+  var r = '';
```

Comment:
is `r` the result and `s` the place to slice?  Either way, please make the variables clearer.  I know the code above isn't doing any better with `k` and `v` :/